src/main/java/io/strimzi/kafka/bridge/http/HttpBridge.java

```diff
@@ -853,4 +850,17 @@ public void process(RoutingContext routingContext) {
             information(routingContext);
         }
     };
+    
+    static class NotSupportedApi extends HttpOpenApiOperation {
+        public NotSupportedApi(HttpOpenApiOperations operationId) {
+            super(operationId);
+        }
+
+        @Override
+        public void process(RoutingContext routingContext) {
+            HttpBridgeError error = new HttpBridgeError(HttpResponseStatus.GONE.code(), "OpenAPI v2 Swagger not supported");
+            HttpUtils.sendResponse(routingContext, HttpResponseStatus.GONE.code(),
+                BridgeContentType.KAFKA_JSON, JsonUtils.jsonToBytes(error.toJson()));
+        }
+    }
```

Comment:
Is this really related to the metrics?

Reply:
No. This is due to some validation/spotbug error I got after rebasing.

Reply:
Which kind of validation/spotpug error? We don't get any right now in the main. This refactoring seems to be useless from a pure code perspective.